src/wasm-lib/kcl/src/settings/types/mod.rs

```diff
@@ -278,8 +278,65 @@ pub struct ModelingSettings {
     /// Whether or not to show a scale grid in the 3D modeling view
     #[serde(default, alias = "showScaleGrid", skip_serializing_if = "is_default")]
     pub show_scale_grid: bool,
+    /// Settings that affect the behavior of the command bar.
+    #[serde(default, alias = "namedViews")]
+    #[validate(nested)]
+    pub named_views: Vec<NamedView>,
```

Comment:
considering these are views id put these in app not modeling settings, i really only want modeling settings to be shit that modifies the engine

Reply:
yes i know some others are in the wrong place but backwards compatibility blah blah blah

Reply:
are these truly app wide or only project wide? only asking since if project only we only need to edit that struct

Reply:
judging by the video demo you only need to edit project types not the global types :)

Reply:
global types are for app-wide settings which these seem not to be, but i would still put this under AppProjectSettings versus Modeling

Reply:
Moved to app settings.

I talked about this with Frank, I am fine with it being moved out of modeling. These are more like presets which I'll put under `app`.